jwcrypto/jwt.py

```diff
@@ -388,6 +399,18 @@ def expected_type(self, v):
         else:
             raise ValueError("Invalid value, must be 'JWS' or 'JWE'")
 
+    def _check_and_convert_dt(self, dt, claim_prop):
+        if (
+            dt.tzinfo is not None
+            and dt.tzinfo.utcoffset(dt) is not None
+        ):
+            dt_timestamp = int(dt.timestamp())
+            return dt_timestamp
+        else:
+            raise ValueError(
+                f"'{claim_prop}' datetime object must be timezone aware"
```

Comment:
Why does it need to be timezone-aware? Can't we just assume UTC if no timezone is set?